src/server/game/Entities/AreaTrigger/AreaTrigger.cpp

```diff
@@ -441,6 +464,29 @@ void AreaTrigger::SetScaleCurve(UF::MutableFieldReference<UF::ScaleCurve, false>
     }
 }
 
+void AreaTrigger::SetOverrideScaleCurve(uint32 startTimeOffSet, std::array<DBCPosition2D, 2> points, CurveInterpolationMode mode /*= CurveInterpolationMode::Linear*/)
+{
+    AreaTriggerScaleCurveTemplate curveTemplate;
+    curveTemplate.StartTimeOffset = startTimeOffSet;
```

Comment:
Is there a situation where startTimeOffSet is not GetTimeSinceCreated ?

Reply:
No clue tbh, but I rather not force it, if you can just set it. I mean it’s not like we have mass sniffs

Reply:
Maybe allow it as optional then, with GetTimeSinceCreated as default